packages/lambda-shared/src/aws/__test__/tile.metadata.table.test.ts

```diff
@@ -62,4 +70,76 @@ o.spec('tile.metadata.table', () => {
             }),
         });
     });
+
+    o('should report missing imagery', async () => {
+        const tmtable = new TileMetadataTable();
+        (tmtable as any).fetchImagery = async (): Promise<any[]> => {
```

Comment:
I think you can drop this any casts, if you make the Promise<any[]> actually the same as the `fetchImagery` return.

Reply:
I need to cast to any because `fetchImagery` is private